### PR TITLE
Add payment highway merchant backend adapter demo lambdas

### PR DESCRIPTION
**Pull request by stepiro**

### CODE REVIEW

File: ph_tokenization_handler.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { buildResponse } from './util';
+
+export const tokenization = async (event, context, callback) => {
+    return executeTokenization(event);
+};
+        
+const transactionTokenResponse = (token) => {
+    if (!token) return {}
+    return {
+        token: token
```
Comment (mendrik) [inline ph_tokenization_handler.js]: just `return { token }`

File: ph_transaction_handler.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import responseJson from './responseJson';
+import getPaymentApi from './phapi';
+import { buildResponse } from './util';
+
+export const transactionId = async (event, context, callback) => {
+  return executeTransactinId();
+};
+ 
+const transactionIdResponse = (id) => {
```
Comment (mendrik) [inline ph_transaction_handler.js]: `const transactionIdResponse = (id) => ({id})`

File: ph_tokenization_handler.js (hunk 1 above)
Comment (mendrik) [reply ph_tokenization_handler.js]: const transactionTokenResponse = (token) => tolen ? {token} : {};

File: phapi.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    phServiceUrl,
+    phKey,
+    phSecret,
+    phAccount,
+    phMerchant
+  );
+};
+
+export const isResponse = (response) => {
+	return (response && response.result && response.result.code) ? true : false;
```
Comment (mendrik) [inline phapi.js]: skip the `? true : false`

File: util.js [hunk 4]
```diff
@@ -1,9 +1,29 @@
...
 }
 
-export default parseJson;
\ No newline at end of file
+export const buildResponse = (phResponse, ...objs) => {
+
+    if (isResponse(phResponse)) {
+      var result = { result: phResponse.result };   
+      if (isSuccess(phResponse)) {
+        for (var obj of objs) {
```
Comment (mendrik) [inline util.js]: replace the for-loop with `Object.assign(result, ...objs)`

File: ph_transaction_handler.js (hunk 2 above)
Comment (stepiro) [reply ph_transaction_handler.js]: thanks, fixed!

File: phapi.js (hunk 3 above)
Comment (stepiro) [reply phapi.js]: thanks!

File: util.js [hunk 5]
```diff
@@ -1,9 +1,29 @@
...
+          result = Object.assign(result, obj)
+        }
+      }
+      return result;
+    }
+
+    return {
+      result: {
+        code: 999,
+        message: "Unknown error!!!"
```
Comment (mendrik) [inline util.js]: this is pretty useless

File: ph_tokenization_handler.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import responseJson from './responseJson';
+import getPaymentApi from './phapi';
+import { buildResponse } from './util';
+
+export const tokenization = async (event, context, callback) => {
+    return executeTokenization(event);
+};
+        
+const transactionTokenResponse = (token) => {
```
Comment (mendrik) [inline ph_tokenization_handler.js]: `const transactionTokenResponse = (token) => token ? { token } : {};`

File: util.js (hunk 4 above)
Comment (stepiro) [reply util.js]: fixed

File: util.js (hunk 5 above)
Comment (stepiro) [reply util.js]: fixed! thanks

File: ph_tokenization_handler.js (hunk 6 above)
Comment (stepiro) [reply ph_tokenization_handler.js]: fixed.

**Pull request merged**